syntax = "proto3";

package com.centreon.broker;

message RemoveGraphMessage {
  repeated uint64 index_ids = 1;
  repeated uint64 metric_ids = 2;
}
